app/services/data_exports/csv/invoice_fees.rb

```diff
@@ -76,15 +68,10 @@ def call
             end
           end
         end
-
-        output.rewind
```

Comment:
isn't needed anymore to rewind the csv output?

Reply:
there's is no more Tempfile, it's using CSV::Generate which runs in memory. (it's a trade-off), but I imagined it would be fine with a low batch size.